src/server/verification/verificationAPI.js

```diff
@@ -190,7 +189,7 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
         const { result = [] } = await fuseapi.getTxList({
           address: user.gdAddress,
           page: 1,
-          offset: 0,
+          offset: 10,
```

Comment:
whats this?

Reply:
it is strange. but offset in this api means "limit", amount of page. if we set 0 it returns all transactions. for ex. admin wallet returns 5400+ transactinos

Reply:
when I set 10. it returns only 10 transactions

Reply:
![image](https://user-images.githubusercontent.com/36693645/64254388-f73aa280-cf27-11e9-906d-0b8a6bb7a827.png)